src/main/webapp/index.html

```diff
@@ -0,0 +1,176 @@
+
+    <!-- Custom -->
+    <link rel="stylesheet" href="css/home.css" />
+    <script src="js/index.js"></script>
+
+    <title>Home</title>
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row ml-1 mt-2">
+        <div class="col-9">
+          <div class="row">
+            You are signed in as test@test.com
+            <a href="#" class="ml-2 btn btn-info">
+              <span class="pull-right glyphicon glyphicon-log-out"></span> Log out
+            </a>
+          </div>
+        </div>
+        <div class="col-3">
+          <a href="#" class="btn btn-info btn-sm float-right">
+            <span class="glyphicon glyphicon-log-out"></span> Add a receipt
+          </a>
+        </div>
+      </div>
+      <h1 class="text-center">Home</h1>
+      <form>
+        <div class="row mb-4">
+          <div class="col-md-4">
+            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search" />
+          </div>
+          <div class="col-md-4">
+            <div id="reportrange" class="time-range-form">
+              <i class="fa fa-calendar"></i>
+              <span></span> <i class="fa fa-caret-down"></i>
+            </div>
+          </div>
+          <div class="col-md-4">
+            <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+          </div>
+        </div>
+      </form>
+      <!-- <p>The columns will automatically stack on top of each other when the screen is less than 768px wide.</p> -->
+      <div class="row">
+        <div class="col-md-6">
+          <div class="card mb-4 box-shadow">
+            <p class="card-text align-self-end">February 14, 2003, 9:46 AM</p>
+            <p class="card-text">Walmart</p>
+            <p class="card-text">Total: $26.12</p>
+            <div class="row">
+              <div class="col-4 d-flex justify-content-center">
+                <h4><span class="badge badge-pill badge-secondary">Candy</span></h4>
+              </div>
+              <div class="col-4 d-flex justify-content-center">
+                <h4><span class="badge badge-pill badge-secondary">Drink</span></h4>
+              </div>
+              <div class="col-4 d-flex justify-content-center">
+                <h4><span class="badge badge-pill badge-secondary">Personal</span></h4>
+              </div>
+            </div>
+            <img src="img/walmart-receipt.jpg" alt="Receipt image" class="img-fluid receipt-img" />
+            <div class="card-body">
+              <div class="d-flex justify-content-between align-items-center">
+                <div class="btn-group">
+                  <button type="button" class="btn btn-sm btn-outline-secondary">View</button
+                  ><button type="button" class="btn btn-sm btn-outline-secondary">Edit</button
+                  ><button type="button" class="btn btn-sm btn-outline-secondary">Delete</button>
```

Comment:
This would be easier to read if the closing `>` was on the same line as the `</button`. Same for lines 137-139 and 165-167.